data/json/itemgroups/vending_machines.json

```
@@ -130,5 +130,303 @@
     "subtype": "collection",
     "id": "vending_medicine",
     "entries": [ { "group": "drugs_vending", "count-min": 10, "count-max": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_adult",
+    "items": [
+      { "item": "condom", "prob": 30, "count": [ 1, 5 ] },
+      { "group": "ss_merch_1", "prob": 20 },
+      { "group": "ss_merch_2", "prob": 20 },
+      { "group": "ss_merch_3", "prob": 20 },
+      { "item": "mag_porn", "prob": 20, "count": [ 1, 3 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_useful",
+    "items": [
+      { "item": "umbrella", "prob": 10 },
+      { "item": "teleumbrella", "prob": 10 },
+      { "group": "home_hw", "prob": 50 },
+      { "group": "tools_common", "prob": 50 },
+      { "group": "cleaning", "prob": 30 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_pet",
+    "items": [
+      { "group": "petstore_shelves", "prob": 100 },
+      { "group": "petstore_misc", "prob": 100 },
+      { "group": "animalshelter_toys", "prob": 100 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_electronics",
+    "items": [
+      { "group": "consumer_electronics", "prob": 40 },
+      { "item": "light_battery_cell", "prob": 20 },
+      { "item": "medium_battery_cell", "prob": 20 },
+      { "item": "heavy_battery_cell", "prob": 10 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "vending_cars",
+    "entries": [
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 },
+      { "group": "supplies_mechanics", "prob": 100 }
```

I'm not even sure what to do about this particular vending machine since it's gonna be loaded with huge bulky stuff that will make little sense being in a vending machine, but that's seemingly the point of it? Maybe small spare parts would be more reasonable?
```suggestion
      { "group": "gloves_work", "prob": 100, "count": [ 1, 5 ] },
      { "item": "wrench", "prob": 50 },
      { "item": "jumper_cable", "prob": 50 },
      { "item": "screwdriver_set", "prob": 50 },
      { "group": "supplies_spares_vehicle", "prob": 100 },
      { "group": "supplies_spares_vehicle", "prob": 100 },
      { "group": "supplies_spares_vehicle", "prob": 100 }
```
I'd use `tools_mechanic` except annoyingly they have damn near every kinda mechanics tools in them EXCEPT wrenches.

---

the idea here was that some vending machines are not uniform and some are larger, although admittedly thinking a muffler-size part could come out of any vending machine (even if it is specialized) is pretty comical